pandas/core/window/rolling.py

```diff
@@ -159,6 +165,8 @@ def validate(self) -> None:
                     f"{type(self.window).__name__} does not implement "
                     f"the correct signature for get_window_bounds"
                 )
+        if self.method not in ["table", "single"]:
+            raise ValueError("method must be 'table' or 'single")
```

Comment:
test for this?

Reply:
Yup, here: https://github.com/pandas-dev/pandas/pull/38417/files#diff-bb790c3c80d044d705ca2b28ecc69cef0460f1cce81492328dda28931b48495cR1128